### PR TITLE
Feat/single movie page

### LINKED ISSUE #2: As a user, I can click a movie, and see that movie’s details

### PR DESCRIPTION
**Pull request by EGavinG**
# Pull Request

## Description
This PR adds in the functionality of Iteration 2. Creates a single movie view page when a movie card is clicked. Showcasing the movie data in detail.

## Changes
Adds a single view page for a selected movie showcasing description:
- Movie Title
- Movie Tagline
- Movie Description
- Movie Release Date
- Movie Rating
- Movie Genre
- Movie Runtime

## Related Issues
- Addresses Issue #2 

## Checklist
- [x] I have read the contribution guidelines.
- [x] I have tested the changes locally.
- [x] The code follows the project's coding standards.
- [x] Documentation has been updated (if applicable).
- [x] I have added/updated unit tests (if applicable).
- [x] I have squashed/organized my commits.

## Additional Notes
None.

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -30,10 +51,44 @@ const App = () => {
 
   return (
     <div className='App'>
-      <h1 tabindex='0'>Rancid Tomatillos</h1>
-      <Carousel movies={popularMovies} badge='Popular' />
-      <Carousel movies={recommendedMovies} badge='Recommended' />
-      <AllMovies movies={allMovies} badge='All' />
+      {!movie && <h1 tabindex='0'>Rancid Tomatillos</h1>}
```
Comment (camjohnson-code) [inline src/Components/App/App.js]: tabindex -> tabIndex

File: src/Components/Card/Card.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import './Card.css';
 
-const Card = ({ averageRating, backdropPath, id, key, releaseDate, title }) => {
+const Card = ({ averageRating, backdropPath, id, key, releaseDate, title, setMovie, dummyMovie }) => {
```
Comment (camjohnson-code) [inline src/Components/Card/Card.js]: We can delete the key prop here

File: src/Components/Card/Card.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ const Card = ({ averageRating, backdropPath, id, key, releaseDate, title }) => {
   };
 
   return (
-    <div className='card'>
+    <div className='card' onClick={() => setMovie(dummyMovie)}>
       <div style={cardStyle} className='poster'></div>
       <p className='title' tabindex='0'>
```
Comment (camjohnson-code) [inline src/Components/Card/Card.js]: tabindex -> tabIndex

File: src/Components/Carousel/Carousel.js
Comment (camjohnson-code) [inline src/Components/Carousel/Carousel.js:1]: Line 40: tabindex -> tabIndex
Line 54: deviceType -> devicetype

File: src/Components/Movies Display/All-Movies.js [hunk 4]
```diff
@@ -14,6 +14,8 @@ const AllMovies = ({ movies, badge }) => {
       releaseDate={movie.release_date}
       title={movie.title}
       tabindex='0'
```
Comment (camjohnson-code) [inline src/Components/Movies Display/All-Movies.js]: tabindex -> tabIndex

File: src/Components/Movies Display/All-Movies.js
Comment (camjohnson-code) [inline src/Components/Movies Display/All-Movies.js:1]: Line 39: deviceType -> devicetype
Line 42 and 43: class -> className

**Review by camjohnson-code — CHANGES_REQUESTED** [review]
Hey! I'm noticing a few typos we made. Once these are done, double check the console to see if there are any errors popping up. I didn't see any once I tweaked these on my end, so you should be good, too! I left comments in App.js, Card.js, Carousel.js and All-Movies.js. If you don't see them, let me know!

**Review by EGavinG — COMMENTED** [review]
Great catch! I'll review the changes and check dev tools on my end to see the changes populate and to ensure no errors.

**Review by EGavinG — COMMENTED** [review]
All clear, fixed all typos and addressed new one with <card> changing it to a <div>

**Review by camjohnson-code — APPROVED** [review]
Looks good! Thanks for helping with those typos!

**Pull request merged**